ports/unix/Makefile

```diff
@@ -107,11 +107,11 @@ endif
 ifeq ($(MICROPY_PY_FFI),1)
 
 ifeq ($(MICROPY_STANDALONE),1)
-LIBFFI_CFLAGS_MOD := -I$(shell ls -1d $(TOP)/lib/libffi/build_dir/out/lib/libffi-*/include)
+LIBFFI_CFLAGS_MOD := -I$(shell ls -1d $(BUILD)/libffi/out/lib/libffi-*/include)
```

Comment:
It would be good to use `$(BUILD)/lib/libffi/...` (ie add `lib/` into the path) because that will follow all other libraries that are build from the root lib directory (eg axtls, berkeley-db-1.xx, etc).

Reply:
Done.